modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -50,6 +53,37 @@ public KafkaContainer(String confluentPlatformVersion) {
         withEnv("KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS", "0");
     }
 
+    @Override
+    public KafkaContainer withNetwork(Network network) {
```

Comment:
Don't we need to remove line 39 in the constructor for this PR to make sense? It will always set `implicitNetwork` to false.

Reply:
thanks for spotting! This is a leftover after splitting one changeset into two :D